fab/fabfile.py

```diff
@@ -948,6 +950,19 @@ def _migrate():
 
 
 @roles(ROLES_DB_ONLY)
+def _migrations_exist():
+    """
+    Check if there exists database migrations to run
+    """
+    _require_target()
+    with cd(env.code_root):
+        n_migrations = int(sudo(
+            '%(virtualenv_root)s/bin/python manage.py migrate --list | grep "\[ ]" | wc -l' % env)
+        )
+        return n_migrations > 0
```

Comment:
interesting thanks. i think i'd have to create a separate management command in order to do this and then maybe key off the exit code. will put it in the back log to do